token/token.go

```diff
@@ -3,21 +3,27 @@ package token
 import (
 	"context"
 	"crypto/rsa"
+	"crypto/x509"
+	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"sync"
 
 	"github.com/fabric8-services/fabric8-auth/log"
 	"github.com/fabric8-services/fabric8-auth/rest"
 
-	"crypto/x509"
-	"encoding/base64"
-
-	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/goadesign/goa"
 	goajwt "github.com/goadesign/goa/middleware/security/jwt"
 	"github.com/pkg/errors"
-	uuid "github.com/satori/go.uuid"
-	jose "gopkg.in/square/go-jose.v2"
+	"github.com/satori/go.uuid"
+	"gopkg.in/square/go-jose.v2"
+	"time"
+)
+
+const (
+	ServiceAccountID = "8f558668-4db7-4280-8e65-408bcb95f9d9"
```

Comment:
Could we maintain this as part of configuration ?

Reply:
Yeah.. good point. Will move it to configuration.

Reply:
But actually, do we really need to keep it in configuration? We don't really use the service account ID. It's mostly just to fill up the sub claim. WIT will check the "service_accountname" claim.
Is there any use case when will will need configure the auth service account ID?
It's similar to the hardcoded SystemSpace ID in WIT.

Reply:
@alexeykazakov , that's a fair point. I just feel a bit uneasy maintaining hard-coded strings this way ( but configuration doesn't feel like the right place either.. )